README.md

```diff
@@ -77,7 +77,7 @@ With the following software and hardware list you can run all code files present
   ```
  * Page 22 (code snippet 2, line 2): **const reducer = (prev, delta) => prev + delta** Should be **const reducer = (prev, delta) => ({ ...prev, count: prev.count + delta })**
  * Page 23 (code snippet 1, line 3): **{state}** Should be **{state.count}**
-  
+ * Page 20 (Paragraph 1, line 2): **takes a second argument.** should be **takes an argument**. 
```

Comment:
Does this work? cc @saurabhk710 
```suggestion
 * Page 20 (Paragraph 1, line 2): **takes a second argument.** should be **takes an argument, which is the second argument**. 
```

Reply:
I think that works, though in that case I think it should then also be "from useReducer" rather than "in useReducer"

```suggestion
 * Page 20 (Paragraph 1, line 2): **takes a second argument---`initialArg`--in useReducer** should be **takes an argument---`initialArg`--, which is the second argument from useReducer**. 
```